### PR TITLE
Telegarmbot(brokenByMe)

### PR DESCRIPTION
**Pull request by AlexMackevich**

### CODE REVIEW

**Review by vitya771 — APPROVED** [review]

**Review by BulavaKatsyaryna — APPROVED** [review]
``

**Review by NikaLu — APPROVED** [review]

File: homework/java/Mackevich_JHW/telegrambot/telegrambot/src/main/java/by/jrr/telegrambot/config/Mapper.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package by.jrr.telegrambot.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class Mapper {
+
```
Comment (max-java) [inline homework/java/Mackevich_JHW/telegrambot/telegrambot/src/main/java/by/jrr/telegrambot/config/Mapper.java:8]: add @Bean annotation

File: homework/java/Mackevich_JHW/telegrambot/telegrambot/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bot.username=rpaS3_bot
+bot.token=1618028646:AAHvNa67rX_KrYjiQjrhuWqD9dWrLQaia7A
```
Comment (max-java) [inline homework/java/Mackevich_JHW/telegrambot/telegrambot/src/main/resources/application.properties:2]: use encryption to store passwords

File: homework/java/Mackevich_JHW/telegrambot/telegrambot/telegrambot.iml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/resources" type="java-test-resource" />
+      <excludeFolder url="file://$MODULE_DIR$/target" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (max-java) [inline homework/java/Mackevich_JHW/telegrambot/telegrambot/telegrambot.iml:16]: add *.iml file to .gitignore

File: homework/java/Mackevich_JHW/telegrambot/telegrambot/src/main/java/by/jrr/telegrambot/service/TelegramBot.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    @Value("${bot.token}")
+    private String botToken;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Override
+    public void onUpdateReceived(Update update) {
+        try {
+           objectMapper.writeValue(new File("src/test/resources/update.json"),update);
```
Comment (max-java) [inline homework/java/Mackevich_JHW/telegrambot/telegrambot/src/main/java/by/jrr/telegrambot/service/TelegramBot.java:28]: use sonar !

**Review by max-java — COMMENTED** [review]
remove target directory from git,
add it to .gitignore

**Pull request merged**